TSPL.docc/LanguageGuide/OpaqueTypes.md

```diff
@@ -921,6 +921,63 @@ which means that the type of `twelve` is also inferred to be `Int`.
   }
 -->
 
+
+## Opaque Parameter Types
+
+In addition to writing `some` to return an opaque type,
+you can also write `some` in the type for a parameter
```

Comment:
Maybe that's a bigger change than what you're after here, but I would introduce `some` parameters first, before `some` returns maybe.